client/src/components/app.jsx

```diff
@@ -21,23 +21,22 @@ class App extends React.Component {
 
   componentDidMount() {
     let id = window.location.pathname;
-    (id === '/') ? id = 1 : id; //revise
+    (id === '/') ? id = 1 : id;
     $.get(`http://localhost:3001/houses${id}`, (data) => {
-      let house = data[0];
+      let { street, city, state, zipcode, description } = data[0];
```

Comment:
Great suggestion. Just implemented this. Knexjs has a method called, ".first()", and allows to return an object rather than an array containing a single object.